content/exporters/custom-exporter/Java/Metrics.md

```diff
@@ -0,0 +1,297 @@
+---
+title: "Metrics exporter"
+draft: false
+weight: 3
+aliases: [/custom_exporter/java/stats, /guides/custom_exporter/java/stats]
+---
+
+- [Introduction](#introduction)
+- [Implementation](#implementation)
+- [Runnable example](#runnable-example)
+- [References](#references)
+
+## Introduction
+
+A metrics exporter must extend the abstract class [MetricExporter](https://static.javadoc.io/io.opencensus/opencensus-exporter-metrics-util/0.19.2/io/opencensus/exporter/metrics/util/MetricExporter.html) implementing the `export` method:
+which for purposes of brevity is:
+
+```java
+import java.util.Collection;
+
+import io.opencensus.exporter.metrics.util.MetricExporter;
+
+public void export(Collection<Metric> metrics);
+```
+
+The sole method `export` will be used to process and translate a collection of [Metric](https://static.javadoc.io/io.opencensus/opencensus-api/0.19.2/io/opencensus/metrics/export/Metric.html) to your desired monitoring backend's data.
+
+## Implementation
+The [Metric](https://static.javadoc.io/io.opencensus/opencensus-api/0.19.2/io/openc\
```

Comment:
This URL results in a 404 due to the `openc\ensus` please don't break URLs
<img width="862" alt="screen shot 2019-02-26 at 2 03 52 am" src="https://user-images.githubusercontent.com/4898263/53379750-25cd3480-396b-11e9-8550-2d4e39b15b07.png">

Reply:
Changed as suggested in new push